crmprtd/moti/normalize.py

```diff
@@ -28,6 +28,9 @@ def normalize(iterable):
     et = transform(et)
 
     obs_series = et.xpath("//observation-series")
+    if len(obs_series) == 0:
+        log.warning("Empty observation series: xpath search "
+                    "'//observation-series' return no results")
```

Comment:
Two things here.

1. The output of `et.xpath("//observation-series")` is a list (or some kind of iterable) which means we can very easily check if it has content directly.  Using `if not obs_series` will return `True` when the iterable has no items (example below). This is simply a styling suggestion so please feel free to take it or leave it.  The code is understandable as is.

```python3
>>> obs_series = []
>>> if not obs_series:
...     print('warning')
... 
warning
```
2. If there are no observations, there is no data to process in the loop below. We would want to exit this method is the condition is triggered.

Reply:
Which exit command should I use?

Reply:
I think you can just use `return`.  Using an empty `return` is the same as returning `None`.  You may want to test this a bit. I imagine there may be some work to do in `process.py` if `None` is returned rather than a generator.